src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -2150,12 +2155,14 @@ private void ClassifyFieldInsert(BaseFieldDeclarationSyntax field)
 
             private void ClassifyFieldInsert(VariableDeclaratorSyntax fieldVariable)
             {
+                RoslynDebug.Assert(fieldVariable.Parent is object);
```

Comment:
>RoslynDebug.Assert(fieldVariable.Parent is object); [](start = 16, length = 51)

I think it'd be better to consistently use Parent! rather than adding asserts in some places and ! in others.